nx-X11/programs/Xserver/mi/miexpose.c

```diff
@@ -762,15 +762,18 @@ int what;
 	if (screenContext[i] == (GCPtr)NULL)
 	{
 	    if (!ResType && !(ResType = CreateNewResourceType(tossGC)))
+		free(prect);
```

Comment:
could be correct, but function is too big to review that now.

Reply:
removed